packages/inline-video/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface InlineVideoProps {
+  appearance?: 'light' | 'dark'
+  url: string
+  description?: string
+  solution?: 'infrastructure' | 'security' | 'networking' | 'applications'
+  gradientSide?: 'left' | 'right'
```

Comment:
💭 How about `gradientPosition` vs `gradientSide`?

Reply:
I like it! Addressed here:
https://github.com/hashicorp/react-components/pull/789/commits/3ad996e88610cc6d7b68231f208620674a254ba1